docs/tutorials/morpheusvm/morpheusvm.md

```diff
@@ -107,8 +107,13 @@ go get github.com/ava-labs/avalanchego && go mod tidy
 ## Creating `Transfer` Pt. 1
 
 We'll implement a single `Transfer` action to handle payments in a native token.
-Actions are wrapped into HyperSDK transactions and executed on-chain, so the struct
-itself will include all of the fields needed to verify and execute onchain.
+Actions are wrapped into the HyperSDK `chain.Transaction` type and executed on-chain.
```

Comment:
Request to provide more detail on how actions are wrapped into the HyperSDK transaction type.